release/cloudbuild_withsecure_signature.yaml

```diff
@@ -61,17 +61,19 @@ steps:
       - call
       - integrate
       - --data
-      - '{"origin": "${_ORIGIN}", "bucket": "${_LOG_NAME}", "kmsKeyName": "ft-log-ci", "kmsKeyRing": "firmware-release-ci", "kmsKeyVersion": 1, "kmsKeyLocation": "global", "noteKeyName": "transparency.dev-aw-ftlog-ci"}'
+      - '{"origin": "${_ORIGIN}", "bucket": "${_LOG_NAME}", "kmsKeyName": "ft-log-ci", "kmsKeyRing": "firmware-release-ci", "kmsKeyVersion": ${KEY_VERSION}, "kmsKeyLocation": "global", "noteKeyName": "transparency.dev-aw-ftlog-ci"}'
 substitutions:
   # TODO(jayhou): do not use CI bucket when we flip this trigger to prod.
-  _FIRMWARE_BUCKET: armored-witness-firmware-ci
+  _FIRMWARE_BUCKET: armored-witness-firmware-ci-1
   _FIRMWARE_COMPONENT: trusted-os
   _WITHSECURE_DIR: release/withsecure
   # TODO(jayhou): remove this when we flip this trigger to prod.
   _TEST_TAG_NAME: withsecure_v0.1.2
   # Log-related.
   _ENTRIES_DIR: firmware-log-sequence
   # TODO(jayhou): switch to prod origin.
-  _ORIGIN: transparency.dev/armored-witness/firmware_transparency/ci/0
+  _ORIGIN: transparency.dev/armored-witness/firmware_transparency/ci/1
   # TODO(jayhou): do not use CI bucket when we flip this trigger to prod.
-  _LOG_NAME: armored-witness-firmware-log-ci
\ No newline at end of file
+  _LOG_NAME: armored-witness-firmware-log-ci-1
+  # This must correspond with the trailing number on the _FIRMWARE_BUCKET, _ORIGIN, _LOG_NAME values.
```

Comment:
Would it be worth breaking up _LOG_NAME and _ORIGIN, and _FIRMWARE_BUCKET to drop the `-1` and just replace them with `{_LOG_NAME}-{_KEY_VERSION}`?

Reply:
Does that work inside the `substitutions` section?

Reply:
Ah, you mean where these subs are used...